sentry.conf.py

```diff
@@ -259,11 +259,18 @@
 SENTRY_WEB_HOST = '0.0.0.0'
 SENTRY_WEB_PORT = 9000
 SENTRY_WEB_OPTIONS = {
+    'http': '%s:%s' % (SENTRY_WEB_HOST, SENTRY_WEB_PORT),
+    'protocol': 'uwsgi',
+    # This is need to prevent https://git.io/fj7Lw
+    'uwsgi-socket': None,
+    'http-keepalive': True,
+    'add-header': 'Connection: Keep-Alive',
+    'memory-report': False,
+    'static-gzip-all': True,
```

Comment:
What's this do and how does it apply to us? We serve static files from the python app. And there's already logic there to serve pre-gzipped files if they exist, but I don't think we pre-gzip them at build time for anything on-premise.

Reply:
Ah, my bad. I got confused about the serving logic. Will remove.